pkgs/tools/misc/netbootxyz-efi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "netboot.xyz-efi";
+  version = "2.0.53";
+
+  src = fetchurl {
```

Comment:
this could probably be the entire derivation, doesn't look like there's a need for a mkDerivation wrapper around the fetch

Reply:
Fixed :+1: